docs: Reactive FeedView updates (#2577)
unoplatform/uno.extensions: doc/Learn/Mvux/FeedView.md

### NoneTemplate

> Setting a template to this property will show when the data returned from the service contains no entries. For instance, if an `IFeed<T>` completed its request successfully with the server returning a `null` result, it's important to note that this is not considered an `Error`. Instead, it's still considered a successful result with no data. Similarly, when using `IListFeed<T>`, the `NoneTemplate` will also display if the collection is empty, as well as if the result is `null.`

Reviewer: ```suggestion
Setting a template to this property will show when the data returned from the service contains no entries. For instance, if an `IFeed<T>` completed its request successfully with the server returning a `null` result, it's important to note that this is not considered an error. Instead, it's still considered a successful result with no data. Similarly, when using `IListFeed<T>`, the `NoneTemplate` will also display if the collection is empty, or if the result is `null`.
```

### ErrorTemplate

The `FeedView` will display this template if the underlying asynchronous operation throws an Exception.

Example:

```xml
<FeedView ...>
    <DataTemplate>
        ...
    </DataTemplate>

    <ErrorTemplate>
        <DataTemplate>
            <TextBlock Text="An error has ocurred while loading the data..." />
        </DataTemplate>
    </ErrorTemplate>
</FeedView>
```

### UndefinedTemplate

> This template is displayed when the `FeedView` loads before the underlying asynchronous operation has even been called.
> As soon as the asynchronous operation is invoked and awaited, the `FeedView` will switch to its `ProgressTemplate` until the operation has resulted in data, at which point it will switch to the `ValueTemplate` or `NoneTemplate`, depending on the data result.

Reviewer: ```suggestion
This template is displayed when the `FeedView` loads before the underlying asynchronous operation has been called.
As soon as the asynchronous operation is invoked and awaited, the `FeedView` will switch to its `ProgressTemplate` until the operation results in data, at which point it will switch to the `ValueTemplate` or `NoneTemplate`, depending on the data result.
```
